llvm/test/tools/llvm-objdump/MachO/AArch64/aliases.s

```diff
@@ -0,0 +1,11 @@
+# RUN: llvm-mc -filetype=obj -triple=arm64-apple-darwin %s -o %t
+# RUN: llvm-objdump --macho -d -M no-aliases %t | FileCheck %s
+# RUN: llvm-objdump --macho -d --disassembler-options=no-aliases %t | FileCheck %s
+
+# CHECK: orr w1, wzr, w2
```

Comment:
What would be the output of this check without the `-M` option?

Reply:
Added a test without -M.